styles.css

```diff
@@ -0,0 +1,114 @@
+body {
+  background-color: #FFD9D9;
+  margin: 60px;
+  max-width: 960;
+}
+
+* {
+  font-family: "century gothic"; 
+}
+
+.header--headline {
+  font-size: 32px;
+  font-weight: bold;
+}
+
+.hr--style {
+  border: 0;
+  height: 1px;
+  background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0));
+}
+
+.main-body--hr__top {
+  border: 0;
+  height: 1px;
+  background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0));
+}
+
+.main-body--hr__bottom{
+  border: 0;
+  height: 1px;
+  background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0));
+  clear: both;
+}
+
+
+.main-body--hr__copy-border
+{ border: 0;
+  height: 1px;
+  background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0));
+}
+
+.main-body {
+  max-width: 960px;
+  
+}
+
+.main-body__portrait{
+  float: left;
+  padding-bottom: 5%;
```

Comment:
You should consider applying padding/margin with em, rem, or even px.  Using percentages may yield unexpected results at some window dimensions.